scripts/pipeline/orthology_benchmark.py

```diff
@@ -544,47 +542,41 @@ def calculate_goc_genes(gene1: str, gene2: str, neighbourhood1: pandas.DataFrame
     strand2 = neighbourhood2.iloc[index2]["strand"]
```

Comment:
There is straight access to the data frame array before checking the existence of indexes in the array. Would a safety sanity check need here?e

Reply:
You're right. I should add safety sanity checks (maybe not just at this place) and then test cases accordingly. That will also address your comment in the test script 🙂